lib/day1.ex

```diff
@@ -56,18 +56,18 @@ defmodule AocKata.Day1 do
   """
   @spec resulting_frequency_fixed(Enumerable.t()) :: integer
   def resulting_frequency_fixed(frequency_changes) do
-    frequency_changes =
-      Stream.repeatedly(fn -> frequency_changes end)
-      |> Stream.flat_map(& &1)
-      |> Stream.cycle()
-      |> Enum.reduce_while({0, %{0 => true}}, fn change, {frequency, reached_frequencies} ->
-        frequency = frequency + String.to_integer(change)
-
-        if reached_frequencies[frequency] do
-          {:halt, frequency}
-        else
-          {:cont, {frequency, Map.put(reached_frequencies, frequency, true)}}
-        end
-      end)
+    frequency_changes = Stream.repeatedly(fn -> frequency_changes end) |> Stream.flat_map(& &1)
+
+    frequency_changes
+    |> Stream.cycle()
```

Comment:
good find - now my thing from line 59 is not needed

Reply:
Basically, my `Stream.repeatedly |> Stream.flat_map` does the same thing as `Stream.cycle`

Reply:
Yepp. Removed it :)